acquisitions/src/main/resources/thunderjet/mod-invoice/invoice-junit.feature

```diff
@@ -8,6 +8,7 @@ Feature: mod-invoice integration tests
       | 'mod-permissions'           |
       | 'mod-users'                 |
       | 'mod-permissions'           |
+      | 'mod-audit'                 |
```

Comment:
this pattern to move tests in mod-invoice is chosen because for order audit test we also do not use cross modules, so just to keep consistency.
I think both options are fine